src/components/charts/bar-chart.tsx

```diff
@@ -59,18 +61,17 @@ const barDatasetDefaults: ChartData<any> = {
   pointHoverBorderWidth: 2,
   pointRadius: 1,
   pointHitRadius: 10,
-  data: [0]
+  data: [0],
+  backgroundColor: ChartColors.map(c => hexToRGBValue(c.hex, 0.4)),
```

Comment:
I'm not sure I quite understand how `backgroundColor` is an array of colors, but maybe it would be easiest to see this in use.